contracts/IColonyNetwork.sol

```diff
@@ -146,19 +151,59 @@ contract IColonyNetwork {
   /// @return resolverAddress Address of the `Resolver` contract
   function getColonyVersionResolver(uint256 _version) public view returns (address resolverAddress);
 
+  /// @notice Activates recovery mode
+  /// @dev Can only be called by authorised address
+  function stop() public;
```

Comment:
These are actually called differently in the Colony recovery implementation. Also only the trivial stop/start functionality is being refactored whereas the entire recovery review and exit is not.